test/controllers/missions_controller_test.rb

```diff
@@ -1,7 +1,4 @@
 require 'test_helper'
 
 class MissionsControllerTest < ActionDispatch::IntegrationTest
```

Comment:
There are still controller tests missing. This is quite easy to do as well and needed.